File: cdk-from-cfn-testing/expected/bucket/python/BucketStack.py
from aws_cdk import Stack
import aws_cdk as cdk
import aws_cdk.aws_s3 as s3
from constructs import Construct

class BucketStack(Stack):
  def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
    super().__init__(scope, construct_id, **kwargs)

    # Resources
    bucket = s3.CfnBucket(self, 'Bucket',
        )

File: cdk-from-cfn-testing/expected/bucket/java/src/main/java/com/myorg/BucketStack.java
package com.myorg;

import software.constructs.Construct;

import java.util.*;
import software.amazon.awscdk.CfnMapping;
import software.amazon.awscdk.CfnTag;
import software.amazon.awscdk.Stack;
import software.amazon.awscdk.StackProps;

import software.amazon.awscdk.*;
import software.amazon.awscdk.services.s3.*;

class BucketStack extends Stack {
    public BucketStack(final Construct scope, final String id) {
        super(scope, id, null);
    }

    public BucketStack(final Construct scope, final String id, final StackProps props) {
        super(scope, id, props);

        CfnBucket bucket = CfnBucket.Builder.create(this, "Bucket")
                .build();

    }
}
